Cargo.toml

```diff
@@ -54,9 +54,9 @@ features = ["vendored-openssl"]
 provides = "dness"
 conflicts = "dness-musl"
 assets = [
-    ["target/release/dness", "usr/bin/dness", "755"],
-    ["assets/bare-config.toml", "etc/dness/dness.conf", "644"],
-    ["assets/dness.service", "etc/systemd/system/dness.service", "644"],
-    ["assets/dness.timer", "etc/systemd/system/dness.timer", "644"]
+    ["target/release/dness", "/usr/bin/dness", "755"],
+    ["assets/bare-config.toml", "/etc/dness/dness.conf", "644"],
+    ["assets/dness.service", "/etc/systemd/system/dness.service", "644"],
+    ["assets/dness.timer", "/etc/systemd/system/dness.timer", "644"]
 ]
-conf-files = ["etc/dness/dness.conf", "etc/dness/dness.env"]
+conf-files = ["etc/dness/dness.conf", "/etc/dness/dness.env"]
```

Comment:
I believe both of these conf-files paths should be absolute

Reply:
Based on the discussion here: https://github.com/mmstick/cargo-deb/issues/172 only the conf-files should be absolute paths, the other asset paths can be left as relative.

Reply:
Let me know if you need me to get this PR polished up.

Reply:
Sorry not sure what needs to be done, appreciate if you can resolve :)